docs/UserGuide.adoc

```diff
@@ -315,6 +327,7 @@ Format: `clear`
 Exits the program. +
 Format: `exit`
 
+
```

Comment:
is there supposed to be an extra line here?

Reply:
Nope, thanks hawk eye.